test/activesupport7/abstract_unit.rb

```diff
@@ -19,7 +19,10 @@
 Thread.abort_on_exception = true
 
 # Show backtraces for deprecated behavior for quicker cleanup.
-ActiveSupport::Deprecation.debug = true
+if ActiveSupport::Deprecation.respond_to?(:debug)
+  # Rails 7.2 does not have ActiveSupport::Deprecation.debug
```

Comment:
@ohler55 & @Watson1978 , for Rails 7.2 I think you want to call:

```ruby
ActiveSupport.deprecator.debug = true
```

Each module in Rails has its own deprecator instance now: https://github.com/rails/rails/commit/e5af9c298a108469a43758297ab56d12b3f0ddcf